web_app/routes/finder_routes.py

```diff
@@ -4,10 +4,11 @@
 finder_routes = Blueprint("finder_routes", __name__)
 
 @finder_routes.route("/finder", methods=["GET","POST"])
-
 def finder():
-    user_city = "userCity"
-    user_state = "userState" 
+    form_data = dict(request.form)
```

Comment:
when the form passes data to the route, the route needs to access this data at `request.form` which is like a dictionary. the dict has keys of the form input "name" attributes, and values of the inputted form values